maquetado/login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>login</title>
+  <link rel="stylesheet" type="text/css" href="application.css">
+</head>
+<body>
+  <div class="login-container">
+    <div class="login">
+      <div class="login-logo-container">
+        <img class="login-logo" src="images/LogoWolox.png" alt="logo">
+      </div>
+      <form class="text-roboto">
+        <div class="login-form-text">
+          Email:
+        </div>
+        <input class="login-form-email-input" type="text">
```

Comment:
You could have a generic input type for your login. They look the same, so you could do `login-input`

Reply:
The difference between the two is simply the bottom margin which is different for both in the invision.